content/docs/command-reference/fetch.md

```diff
@@ -106,6 +107,13 @@ The `dvc remote` used is determined in order, based on
   workspace. This downloads tracked data for the entire commit history of the
   project.
 
+- `--max-size <bytes>` - fetch data files/directories that are each below
+  specified size (bytes). The size is determined by a corresponding `size` field
+  in the `.dvc`/`dvc.lock` file.
+
+-- `--type <type>` - fetch data files/directories that are of a particular type.
```

Comment:
what happens if I have a single plot file inside a DVC-tracked directory? will it fetch only that single file?

Reply:
It will fetch the whole directory, output-level (aka data level) granularity is the finest we currently support in CLI.

Reply:
and if doesn't pass the threshold (the whole directory), it won't fetch anything ...

not sure how important is this ... but it all can be at least a bit unexpected for users who would deal with this

> granularity is the finest we currently support in CLI.

btw, I though we had a way to fetch a specific file(s) if needed w/o the whole output, or do you mean this specific CLI option only?

Reply:
Yes, you can fetch and even pull. I guess I should not have said "the finest we currently support in CLI", as it is not quite true and it is rather a mixed bag.